lib/index.js

```diff
@@ -140,6 +140,14 @@ function getOptions(options) {
   options.sourceMap = getSourceMap(options);
   options.style = getStyle(options) || 0;
 
+  // context object represents node-sass environment
+  options.context = {
+    info: module.exports.info,
+    state: {},
+    sync: false,
```

Comment:
That totally makes sense. In ruby sass, there is a `custom` element and developers are told not to write to the top level hash. http://sass-lang.com/documentation/file.SASS_REFERENCE.html Should we instead rename this to `custom` for consistency? I do think there's a lot of value in people not writing into the top level hash or the option hash.